src/rest_api/classes/expr_pattern/core.clj

```diff
@@ -6,14 +6,25 @@
 
 (defn pack [ep ep-holder]
   (let [h (first (:expr-pattern/gene ep))
-	gene (:expr-pattern.gene/gene h)]
+        gene (:expr-pattern.gene/gene h)]
     {:certainty (generic-functions/certainty ep-holder)
      :expression_pattern (pack-obj ep)
      :reference (some->> (:expr-pattern/reference ep)
-			 (map :expr-pattern.reference/paper)
+                         (map :expr-pattern.reference/paper)
                          (map pack-obj))
      :gene (some->> (:expr-pattern/gene ep)
                     (map :expr-pattern.gene/gene)
                     (map pack-obj))
      :author (:author/id (first (:expr-pattern/author ep)))
-     :description (first (:expr-pattern/pattern ep))}))
+     :description (or
+                    (first (:expr-pattern/pattern ep))
+                    (or
+                      (some->> (:expr-pattern/subcellular-localization ep)
+                               (sort)
+                               (str/join "<br />")
+                               (not-empty))
+                      (some->> (:expr-pattern/remark ep)
+                               (map :expr-pattern.remark/text)
+                               (sort)
+                               (str/join "<br />")
+                               (not-empty))))}))
```

Comment:
Not sure if `(not-empty)` works on a string. It takes a collection as an input. But if you think it's fine, then it's okay.